pivy-importer/src/main/groovy/com/linkedin/python/importer/ivy/IvyFileWriter.groovy

```diff
@@ -98,6 +99,10 @@ class IvyFileWriter {
         return filename.substring(filename.indexOf(version) + version.length() + 1)
     }
 
+    static String getActualModuleName(String filename, String revision) {
+        return filename.substring(0, filename.indexOf(revision) - 1)
```

Comment:
Isn't there already a regex for wheels package names that we can use for this instead of substring?

Reply:
Regex is another possible solution, but here we just need to get the actual module name, I would prefer to use the similar way as how we get classifier (getClassifier), it's more readable.

Reply:
Ah I hadn't seen getClassifier. Ok it makes sense then. Should we make it private and non-static like getClassifier?